inline_scan_docker.sh

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-docker run --rm -ti -v /var/run/docker.sock:/var/run/docker.sock sysdiglabs/secure-inline-scan:2 "$@"
+docker run --rm -ti sysdiglabs/secure-inline-scan:2 "$@"
```

Comment:
Why not mount the docker sock? This script is just intended to be a wrapper for the docker daemon scan use case (like the v1). 
Although I think this does not make sense anymore, we could just remove it. If we want it to be a docker wrapper, we might need to add:

`--storage-type docker-daemon `

to the automatically added parameters.

I would vote for removing this script. The README is clear enough on how to run the container. @ProvoK what do you think?